.github/workflows/scripts/pre-submit.delegators/compare-diff.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cat <<EOF >DIFF
```

Comment:
nit: Put the diff in a .diff file and read the file here. That will make it easier to update it.